docker-compose.yml

```diff
@@ -1,31 +1,136 @@
 version: "3"
 
+# Common build configuration for Airflow
+# Extension field, see https://docs.docker.com/compose/compose-file/compose-file-v3/#extension-fields
+x-airflow-common: &airflow-common
+  profiles:
+    - catalog
+  restart: on-failure
+  depends_on:
+    - postgres
+    - s3
+  image: openverse_catalog
+  env_file:
+    - .env
+    - catalog/.env
+  build:
+    context: ./catalog/
```

Comment:
All of the other images are under `docker/<service>/`, should this be as well? Or is that in a separate folder so the `catalog/docker-compose.yml` file will still work? And if it's the latter, can we add a comment somewhere about that difference?

Reply:
`Dockerfile`s for our core services like API, frontend and catalog are kept with the code folders, and those for services these depend on (like Postgres etc) are kept inside the `docker/` folder.

The `docker-compose.yml` file in the root of the repo is the development one (includes all services grouped by profiles) and each stack that uses Compose for deployment also has a `docker-compose.yml` file inside the codebase (like the `ingestion_server/` in the monorepo).